client/packages/system/src/Stock/Components/InventoryAdjustment/AdjustmentForm.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import {
+  useTranslation,
+  Box,
+  NumericTextInput,
+  AdjustmentTypeInput,
+  getReasonOptionTypes,
+  useAuthContext,
+  StoreModeNodeType,
+  FormLabel,
+} from '@openmsupply-client/common';
+import { DraftInventoryAdjustment } from '../../api';
+import { ReasonOptionsSearchInput } from '../../..';
+import { InventoryAdjustmentDirectionInput } from './InventoryAdjustmentDirectionSearchInput';
+
+export const AdjustmentForm = ({
+  draft,
+  setDraft,
+  isVaccine,
+}: {
+  draft: DraftInventoryAdjustment;
+  setDraft: Dispatch<SetStateAction<DraftInventoryAdjustment>>;
+  isVaccine: boolean;
+}) => {
+  const t = useTranslation();
+  const { store } = useAuthContext();
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '1em',
+        width: '30em',
+        margin: '1.5em auto',
+      }}
+    >
+      <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+        <FormLabel sx={{ fontWeight: 'bold' }} htmlFor="by">
+          {t('label.adjust-by')}
+        </FormLabel>
+        <Box
```

Comment:
It's showing up with no gap between the +/- and the input for me, which is different to your screenshots:

<img width="453" height="67" alt="Image" src="https://github.com/user-attachments/assets/82037348-e3e7-4091-bc16-323f85060be9" />